src/librustdoc/html/layout.rs

```diff
@@ -90,6 +90,7 @@ pub fn render<T: Print, S: Print>(
             <div class=\"search-container\">\
                 <div>{filter_crates}\
                     <input class=\"search-input\" name=\"search\" \
+                           disabled=\"true\" \
```

Comment:
Please put `disabled` at last. Also, no need for `="true"`: https://www.w3schools.com/tags/att_input_disabled.asp

Reply:
An attribute without a value is not valid XHTML. I took the habit of writing valid XHTML so I can get free XML validation by the browser :)
I can remove it if you really don't want it, but it probably doesn't hurt. (You should also change all the `/>`endings in tags in this case, as they are not needed in HTML).

Reply:
The rest isn't XHTML compatible so there's no reason to make it follow this rule here. :p